tailcfg/c2ntypes.go

```diff
@@ -52,3 +52,19 @@ type C2NUpdateResponse struct {
 	// Started indicates whether the update has started.
 	Started bool
 }
+
+// C2NPostureIdentityResponse contains either a set of identifying serial number
+// from the client or a boolean indicating that the machine has opted out of
+// posture collection.
+type C2NPostureIdentityResponse struct {
+	// Err is the error message, if any.
+	Err string `json:",omitempty"`
+
```

Comment:
I can see how eventually we might want to collect errors, aggregate them and show tailnet admins, but perhaps for now it will be sufficient to simply log the error?